Connect/AzureStack.Connect.psm1

```diff
@@ -8,31 +8,49 @@
     .SYNOPSIS
     Registers all providers on the all subscription
 #>
-function Register-AllAzureRmProvidersOnAllSubscriptions {
+
+# Temporary backwards compatibility.  Original name has been deprecated. 
+New-Alias -Name 'Register-AllAzureRMProvidersOnAllSubscriptions' -Value 'Register-AzsProviderOnAllSubscription' -ErrorAction SilentlyContinue
+
+function Register-AzsProviderOnAllSubscription {
+
+    Write-Warning "The function '$($MyInvocation.MyCommand)' is marked for deprecation. Please remove any references in code."
+
     foreach($s in (Get-AzureRmSubscription)) {
-        Select-AzureRmSubscription -SubscriptionId $s.SubscriptionId | Out-Null
-        Write-Progress $($s.SubscriptionId + " : " + $s.SubscriptionName)
-    Register-AllAzureRmProviders
-}
+            Select-AzureRmSubscription -SubscriptionId $s.SubscriptionId | Out-Null
+            Write-Progress $($s.SubscriptionId + " : " + $s.SubscriptionName)
+        Register-AzsProvider
```

Comment:
This command is Azure centric and will be deprecated - not sure we need to rename it